torch_geometric/nn/aggr/multi.py

```diff
@@ -66,6 +66,9 @@ def __init__(
         self.mode = mode
         mode_kwargs = mode_kwargs or {}
         if mode == 'proj' or mode == 'attn':
+            if len(aggrs) == 1:
+                raise ValueError("One single aggregation does not support "
```

Comment:
```suggestion
                raise ValueError("Multiple aggregations required for "
```
I feel is slightly more clear

Reply:
Sounds good.

Reply:
Thanks!